assets/styles/pages/confirmation.less

```diff
@@ -190,4 +191,71 @@
       }
     }
   }
-}
\ No newline at end of file
+  .referral-info-container {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 100px;
+    .next-reward {
+      border: 8px solid @color-yellow;
+      border-radius: 50%;
+      height: 300px;
+      margin: 0;
+      position: relative;
+      width: 300px;
+
+      i {
+        background-position: center;
+        background-size: cover;
+        border: 8px solid white;
+        border-radius: 50%;
+        box-sizing: border-box;
+        display: block;
+        filter: brightness(125%);
+        height: 284px;
+        width: 284px;
+      }
+
+      i.reward-image-1 {
+        background-image: url('../images/referrals/referral-reward-1.jpg');
+      }
+
+      .next-reward-count {
+        background: white;
+        border: 8px solid white;
+        border-radius: 50%;
+        bottom: 0;
+        box-sizing: content-box;
+        height: 132px;
+        position: absolute;
+        right: 0;
+        transform: translate3d(25%, 25%, 0);
+        width: 132px;
+
+        div {
+          align-items: center;
+          border: 8px solid #ffcc45;
+          border-radius: 50%;
+          box-sizing: border-box;
+          display: flex;
+          font-size: 12px;
+          font-weight: 700;
+          height: 132px;
+          justify-content: center;
+          padding-top: 12px;
+          width: 132px;
+
+          span {
+            font-size: 50px;
+            margin-top: -12px;
+          }
+        }
+      }
+    }
+    .referral-status-link {
```

Comment:
I think we could also add something like a `max-width: 500px` to keep the text here more compact on desktop-width screens.